### DISCUSSION TITLE
Pass autocomplete only works directly after installing oh-my-zsh

### DISCUSSION THREAD

**Discussion by johnnymatthews** [ohmyzsh/ohmyzsh#10121] [Help] [answered]
I'm running MacOS BigSur M1. I'm using the [Unix Pass manager](https://www.passwordstore.org/), which is used with the `pass` command. The autocomplete bundled with oh-my-zsh doesn't seem to be working properly.

Here's what I'm doing.

1. Open terminal (Hyper, iTerm, whatever; doesn't seem to matter).
2. Install Zsh.
3. Install Oh-My-Zsh.
4. At the point, `pass` auto complete is working fine. So `pass` + `TAB` will list my password options.
5. If I close the terminal and then open it back up again, the auto complete no longer works.

I can uninstall oh-my-zsh with `uninstall_oh_my_zsh`, reinstall it with the `sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`, close the terminal, and then reopen it again, and `pass` autocomplete works again! But then as soon as I close that terminal window, I'll lose the autocomplete.

I've no idea how to even start degugging this. I'm not sure if it's an issue with my installation of ZSH, or something to do with MacOS?

**Comment by adrienafl** [comment, marked as answer]
Add this to .zshrc : 

```
if type brew &>/dev/null; then
	  FPATH=$(brew --prefix)/share/zsh-completions:$(brew --prefix)/share/zsh/site-functions:$FPATH

    autoload -Uz compinit
    compinit
  fi
```

**Reply by muknta** [reply]
> Add this to .zshrc :
> 
> ```
> if type brew &>/dev/null; then
> 	  FPATH=$(brew --prefix)/share/zsh-completions:$(brew --prefix)/share/zsh/site-functions:$FPATH
> 
>     autoload -Uz compinit
>     compinit
>   fi
> ```

~~Doesn't work for me.~~ MacBook Pro M1, Monterey.
Oh-my-zsh already installed.

I have the same ~/.zshrc file(even without your suggested lines) on my MacBook Pro 15 Intel, Monterey, where it works fully.

**UPD: it works after relaunching of terminal :)**

**Reply by adrienafl** [reply]
If I remember well, I noticed the "site-functions" folder was missing to have pass working smoothly. Folder location changes on M1 Mac, that's why :)
